powa--3.2.0--3.2.1dev.sql

```diff
@@ -0,0 +1,314 @@
+-- complain if script is sourced in psql, rather than via CREATE EXTENSION
+--\echo Use "ALTER EXTENSION powa" to load this file. \quit
+
+
+  INSERT INTO powa_functions (module, operation, function_name, added_manually, enabled) VALUES
+      ('pg_stat_statements', 'purge', 'powa_databases_purge', false, true);
```

Comment:
nitpicking, but it's useless leading whitespace :)

Reply:
I don't know what you're talking about muwawawah

Reply:
:)